misk/src/test/kotlin/misk/web/interceptors/TracingInterceptorTest.kt

```diff
@@ -37,6 +38,22 @@ class TracingInterceptorTest {
     assertThat(mockTracer.finishedSpans().size).isEqualTo(1)
   }
 
+  @Test
+  fun failedTrace() {
+    val tracingInterceptor = tracingInterceptorFactory.create(FailedTracingTestAction::call.asAction())!!
+    val chain = failedTracingTestAction.asChain(FailedTracingTestAction::call, emptyList(), tracingInterceptor)
+
+    @Suppress("UNCHECKED_CAST")
+    chain.proceed(chain.args) as Response<String>
+
+    val mockTracer = tracer as MockTracer
+    assertThat(mockTracer.finishedSpans().size).isEqualTo(1)
+
+    val span = mockTracer.finishedSpans().first()
+    assertThat(span.tags().get(Tags.ERROR.key)).isEqualTo(true)
+    assertThat(span.tags().get(Tags.HTTP_STATUS.key)).isEqualTo(400)
+  }
+
```

Comment:
Add test for a custom exception that gets mapped to a non-successful response via the ``ExceptionMappingInterceptor``